swap_meet/vendor.py

```diff
@@ -1,2 +1,84 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=[]):
+        self.inventory = inventory or []
+        
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
👍 We need to check the item is actually there before using remove, or else we would need to handle the error that could be thrown if we try to delete something that doesn't exist in the list. 

If we reorder the logic we can emphasize the main logic of this method by unindenting it and placing it in a more significant part of the function, rather than being just one branch of a condition.

```suggestion
        if item not in self.inventory:
            return False

        self.inventory.remove(item) 
        return item
```

Reply:
Something to consider: Your function currently returns `item` that was passed into the method, rather than the actual instance of that item that was removed from `inventory`. 

If we really want to be sure that we're returning the item removed from the `inventory`, we could use index to find its location (which would raise an `IndexError` if it wasn't found). Then we could return the result of popping the value. This would give us the actual instance from the `inventory` list.